tests/test_general.py

```diff
@@ -8,19 +11,31 @@
 from uncertainties import unumpy as unp
 import matplotlib.pyplot as plt
 
-# ToDo: specify what error is raised
+
+@pytest.fixture(params=[Hist, BaseHist, NamedHist])
+def named_hist(request):
+    yield request.param
+
+
+@pytest.fixture(params=[Hist, BaseHist])
+def unnamed_hist(request):
+    yield request.param
```

Comment:
@LovelyBuggies Here's how you would parametrize tests. However, since Hist and BaseHist are identical, and BaseHist is not supposed to be user-facing, we should just test Hist in most cases, unless there's a reason to check BaseHist (case in point: `Hist.new` should return a `Hist`, while `BaseHist.new` should return a BaseHist, so that's a good thing to check).